Source/RuntimeFilesDownloader/Private/RuntimeChunkDownloader.cpp

```diff
@@ -2,285 +2,253 @@
 
 #include "RuntimeChunkDownloader.h"
 
+#include "FileFromStorageUploader.h"
```

Comment:
Looks like there are changes in this file but cannot discern what they are as it appears your IDE has reformatted the entire file. Any way you can remove the formatting here and just commit code changes?

Reply:
Done, the easiest way was to squash everything. Sorry now you can't read a diff to what you've already reviewed :/